app/src/main/java/com/nitramite/paketinseuranta/MainMenu.java

```diff
@@ -97,6 +97,7 @@
     private View emptyView;
     private SharedPreferences sharedPreferences;
     private Boolean SP_UPDATE_FAILED_FIRST = false;
+    private boolean lastUpdate = false;
```

Comment:
Rename to `showLastEventDate` for consistency and make it clear it's a boolean and not the date string.